python/test/options.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# options.py
+
+import argparse
+
+import os
+
+"""
+parse_arguments
+    Goal:       define and read arguments given by the user
+    Arguments:  args      - list of strings to parse, default from sys.argv
+                namespace - object to take the attributes, default new empty namespace object
+    Returns:    attributes of namespace
+    Example:    args = parse_arguments() 
+"""
+def parse_arguments(args=None, namespace=None):
+    parser = argparse.ArgumentParser()
+
+    # add arguments as needed (e.g. -d, --dryrun Number of dry run ranks)
+    parser.add_argument("-v", "--verbosity", nargs='?', const=0, type=int, choices=[0, 1, 2], default=0, help="increase output verbosity")
+    #parser.add_argument("-d", "--dryrun", type=int, default=100 , help="number of dry run ranks")
+    args = parser.parse_args()
+    return args
+
+"""
+bool_env
+    Goal:       get an environment variable coerced to a boolean value.
+    Arguments:  var_name  - name of the environment variable
+                default   - default to use if `var_name` is not specified in environment
+    Returns:    `var_name` or `default` coerced to a boolean using the following rules
+                False: "False", "false", "OFF", "off", "Off", ""; True: any other non-empty string
```

Comment:
I think we can do without this (see other comments), but I prefer to keep it simple
```
# turn on MPI
export ARB_WITH_MPI
# turn it off
unset ARB_WITH_MPI
```
to avoid testing all those strings ;)

Reply:
Done. 
* Removed python argument reader, 
* added boolean function (+wrapper) in `mpi.cpp` to check if arbor was compiled with MPI/mpi4py,
* removed `test/runner.py` (so that test are now to be run in regarding foolders `unit`or `unit_distributed` via the respective runners as `python -m runner` instead of `python -m unittest`)